### PR TITLE
docs(addition): add docs for lsd protocol

### PR DESCRIPTION
**Pull request by JafarAz**
Required for merge:
- [ ] `pr-workflow-check / draft-release-check` is ✅ success
- Other rules GitHub shows you, or can be read in [configuration](../terraform/github.com/branches.tf) 

Makes review faster:
- [x] PR title is my best effort to provide summary of changes and has clear text to be part of release notes 
- [x] I marked PR by `misc` label if it should not be in release notes
- [ ] Linked Zenhub/Github/Slack/etc reference if one exists
- [ ] I was clear on what type of deployment required to release my changes (node, runtime, contract, indexer, on chain operation, frontend, infrastructure) if any in PR title or description
- [x] Added reviewer into `Reviewers`
- [ ] I tagged(`@`) or used other form of notification of one person who I think can handle best review of this PR
- [ ] I have proved that PR has no general regressions of relevant features and processes required to release into production
- [ ] Any dependency updates made, was done according guides from relevant dependency
- Clicking all checkboxes 
- Adding detailed description of changes when it feels appropriate (for example when PR is big)

### CODE REVIEW

**Review by RustNinja — APPROVED** [review]

File: docs/docs/technology/liquid-staking.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+In the near future, this can be used to secure Composable’s ecosystem in line with the proof-of-stake (PoS) consensus model. This will be accomplished by allowing the restaking of LSDOT. **The result is pooled security**. This solution can be expanded to be offered to apps on Cosmos and other networks as Composable’s IBC connections continues to add new networks (such as to Ethereum and Solana).
+
+Initially, the first step will be to facilitate the restaking of LSDOT into Composable's restaking layer. This restaking layer is vital as it not only allows for securing Composable's network via PoS, but also facilitates partial block building and thus relieves concerns over block proposer censorship and agency issues. Once this is model is implemented on Composable's restaking layer, it can be integrated as a middleware and apps on various blockchains. As a result, it will create a use case for LSDOT in Cosmos and other networks, while delivering pooled security to the Composable ecosystem and beyond.
```
Comment (dzmitry-lahoda) [inline docs/docs/technology/liquid-staking.md:15]: not clear part on block proposer and agency, may be other doc

File: docs/docs/technology/liquid-staking/technical-overview.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+The Liquid Staking pallet residing on Composable Polkadot is responsible for executing functions such as bonding, unbonding, and rebonding with the Polkadot Relay chain. The Relay chain updates validators in every Era, and the Liquid Staking pallet processes user requests and operates in sync with the Relay chain's Era updates. This functionality is facilitated by the MatchingPool module.
+
+:::info
+An Era refers to a (whole) number of sessions, which is the period that the validator set and each validator's active nominator set is recalculated and where rewards are paid out. An era equals one day on the Polkadot Relay chain.
+:::
+
+## DOT Distribution
+
+To optimise APY and risk management, the Liquid Staking pallet utilises multiple derivative accounts on the Polkadot Relay chain, each of which acts as an independent stash account. These accounts are used to distribute funds effectively based on a sophisticated fund allocation algorithm.
```
Comment (dzmitry-lahoda) [inline docs/docs/technology/liquid-staking/technical-overview.md:13]: not clear security implications of this

Comment (dzmitry-lahoda) [reply docs/docs/technology/liquid-staking/technical-overview.md:13]: afaik chain has one account only.  not sure it can create more

Comment (dzmitry-lahoda) [reply docs/docs/technology/liquid-staking/technical-overview.md:13]: unless there is such feature added to derive accou ts on behalf of parachain(afaik composable runtime has such for dao chain)

Comment (RustNinja) [reply docs/docs/technology/liquid-staking/technical-overview.md:13]: @dzmitry-lahoda it uses utility_pallet 
UtilityCall
https://github.com/ComposableFi/composable/blob/liquid-staking/code/parachain/frame/pallet-xcm-helper/src/lib.rs#L330-L349

first it move assets from sovereign account to derivative account.
then bond/nominate/unstake from this derivative account.

File: docs/docs/technology/liquid-staking/why-lsd.md [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+
+### Censorship and Lost Agency
+
+Censorship concerns are another issue on a number of blockchains. This is particularly true on Ethereum since its merge to proof-of-stake (PoS) and the creation of MEV-Boost by Flashbots. Ethereum validators use MEV-Boost to earn a share of the MEV extracted from the blocks they propose to block builders, made possible by proposer-builder separation (PBS) in Ethereum. Using MEV-Boost, validators access blocks from a marketplace of builders who produce blocks with information on the transaction orderflow and fee for the validator who proposes the block. 
+
+A significant downside of MEV-Boost is that it only allows MEV extraction for full blocks, as proposers attest to block headers only when selecting the highest bids. Thus, proposers cannot customize block contents. This makes room for censorship; builders could collude to censor transactions for extortion. Moreover, the full-block nature of MEV-Bost means there is limited slashing capability; slashing can only occur when a block proposer proposes a different block than the committed block (e.g. block proposers sign two block headers of the same height).
```
Comment (dzmitry-lahoda) [inline docs/docs/technology/liquid-staking/why-lsd.md:15]: Bost -> Boost

**Review by dzmitry-lahoda — APPROVED** [review]

**Comment by dzmitry-lahoda** [comment]
why-lsd is hard to read

**Pull request merged**